### PR TITLE
Updated example on fieldset page.

### PR DESCRIPTION
**Pull request by calvinfroedge**
Added example for adding form tag attributes with set_config.  Had to dig through Form class to figure out.  On another note, please read this forum thread about documentation consistency, it seems to me different people are documenting in slightly different ways.  Documentation SHOULD be standardized, right?

http://fuelphp.com/forums/topics/view/2499

### CODE REVIEW

**Comment by calvinfroedge** [comment]
Btw, on that merge, I see lots of changes from others.  Here are commands I used on my machine.  Is any of this wrong?  

git clone git@github.com:calvinfroedge/docs.git
git checkout -b develop
cd classes
vi fieldset.html
git pull origin develop
git push origin develop

File: classes/fieldset.html [hunk 1]
```diff
@@ -392,9 +392,13 @@
 					<td>Fuel\Core\Fieldset Object</td>
 				</tr>
 				<tr>
-					<th>Example</th>
+					<th>Examples</th>
 					<td>
+						<i><code>//General example</code></i>
 						<code>$article_form->set_config($config);</code>
+						<br />
+						<i><code>//Example of setting attributes to form tag with Fieldset class</i></code>
```
Comment (frankdejonge) [inline classes/fieldset.html]: I'll have to look into it to merge, but in the meanwhile you've got an nesting mismatch here...

Comment (calvinfroedge) [inline classes/fieldset.html]: Fixed.  Thanks!

**Comment by frankdejonge** [comment]
Calvin, there were more issues with the html, and many more on the other html on the page itself so at the end it was better to get my own hand dirty and fix it all at once. Thanks for the example, I've included a simular one in the docs.

**Pull request closed without merging**